src/dotnet-releaser/DevHosting/GitHubDevHosting.cs

```diff
@@ -466,6 +468,68 @@ public async Task UploadHomebrewFormula(string user, string repo, ProjectPackage
 
     }
 
+    public async Task UploadScoopManifest(string user, string repo, ProjectPackageInfo projectPackageInfo, string scoopManifest)
+    {
+        var appName = projectPackageInfo.AssemblyName;
+        var filePath = $"bucket/{appName}.json";
+
+        Repository? existingRepository = null;
+        try
+        {
+            existingRepository = await _client.Repository.Get(user, repo);
+        }
+        catch (NotFoundException)
+        {
+            // ignore
+        }
+
+        if (existingRepository is null)
+        {
+            _log.Info($"Creating Scoop repository {user}/{repo}");
+            var newRepositoryFromTemplate = new NewRepositoryFromTemplate(repo)
+            {
+                Description = $"Scoop repository for {projectPackageInfo.ProjectUrl}",
+            };
+            await _client.Repository.Generate("ScoopInstaller", "BucketTemplate", newRepositoryFromTemplate);
+        }
+        else
+        {
+            _log.Info($"Scoop repository found {user}/{repo}");
+        }
+
+        IReadOnlyList<RepositoryContent>? repositoryContents = null;
+        try
+        {
+            repositoryContents = await _client.Repository.Content.GetAllContents(user, repo, filePath);
+        }
+        catch (NotFoundException)
+        {
+            // ignore
+        }
+
+        var shouldCreate = repositoryContents is null || repositoryContents.Count == 0;
+        if (shouldCreate)
+        {
+            _log.Info($"Creating Scoop manifest {user}/{repo}/{filePath}");
+            await _client.Repository.Content.CreateFile(user, repo, filePath, new CreateFileRequest($"{projectPackageInfo.Version}", scoopManifest));
+        }
+        else
+        {
+            Debug.Assert(repositoryContents is not null);
+            var file = repositoryContents[0];
```

Comment:
How is it that the first content is always going to be the manifest?

Reply:
Totally missed that.